firebase/README.md

```diff
@@ -10,24 +10,66 @@ You will need to set up a new firebase project and enable the following features
 - Hosting
 - Functions
 
+Note that you will not need to run `firebase init` as suggested during setup, because the project is already initialised with a .firebaserc file
+
 ## Configuration
 
+## Environment variables
+
+You should create a `.env` file in the project root according to the `.env.tmpl` file. This needs populating with your firebase project ID and API key. Instructions for doing so can be found in the [template file](../.env.tmpl). When building the application, these values will be compiled into the javascript.
+
+(If successful, `npm start` will now get you to an error page within the app)
+
+## Billing
+
+The Firebase functions we use make API calls to slack. As this is an external network request, you will be required to enable billing on your firebase account and change your plan to the Blaze pay-as-you-go pricing plan (or better).
+
+To do this for ScottLogic, you will need to gain access to ScottLogic's Billing account. See Chris Price or Colin Eberhardt for details.
```

Comment:
Can you remove this line? I'd rather not have anything SL specific in here

Reply:
I'd like to put some kind of line in to stop an overenthusiastic user racking up a $1,000 charge and blaming us.

"As with any pricing plan, make sure you get approval from whoever is responsible for paying for any charges?"